admin/preprints/serializers.py

```diff
@@ -18,12 +18,10 @@ def serialize_preprint(preprint):
 
 def serialize_subjects(subject_chains):
     serialized_subjects = []
-    for subject_chain in subject_chains:
-        for subject in subject_chain:
-            subject = Subject.load(subject)
-            if subject:
-                serialized_subjects.append({
-                    'id': subject._id,
-                    'text': subject.text
-                })
+    subject_ids = [_id for subject_chain in subject_chains for _id in subject_chain]
+    for subject in Subject.objects.filter(_id__in=subject_ids):
```

Comment:
Minor: If you only need a few attributes, you can use .only

```python
Subjects.objects.filter(...).only('_id', 'text')
```

https://docs.djangoproject.com/en/1.10/ref/models/querysets/#only

Definitely not a blocker though.